source/includes/_tenants.md

```diff
@@ -266,4 +265,280 @@ Delete a tenant.
 * `tenant-name-to-delete`: The name of the tenant to delete.
 
 ### Response
+A `Tenant` resource.
+
+
+
+
+
+## Add User to Tenant
+
+> Request Example
+
+```shell
+$ curl -X PUT -H "Content-Type: application/json" -H "tenant: <tenant-name>" -u user:password -d '{"username": <user-name>, "tenant_name": <tenant-name>}' `"http://<manager-ip>/api/v3/tenants/users"
+```
+
+```python
+# Python Client-
+client.tenants.add_user(<user-name>, <tenant-name>)
+```
+
+```javascript
+var headers = {
+   'content-type': 'application/json',
+   'authorization': 'Basic ' + new Buffer(username + ':' + password).toString('base64'),
+   'tenant': <tenant-name>
+}
+
+vat data = {
+    "username": <user-name>,
+    "tenant_name": <tenant-name>
+}
+
+var settings = {
+  "url": "http://<manager-ip>/api/v3/tenants/users",
+  "method": "PUT",
+  "headers": headers,
+  "data": data
+}
+
+$.ajax(settings).done(function (response) {
+  console.log(response);
+});
+```
+
+```html
+obsolete
+```
+
+> Response Example
+
+```json
+{
+    "name": "tenant-name",
+    "groups": [],
+    "users": []
+}
+```
+
+`PUT "{manager-ip}/api/v3/tenants/users"`
+
+Add a user to tenant.
+
+### Request Body
+
+Property | Type | Description
+--------- | ------- | -----------
+`username` | string | The user name to add to the tenant.
+`tenants_name` | string | The tenants name to add the user into it.
```

Comment:
~~The tenants name to add the user into it.~~ **The name of the tenant to which to add the user.**